compiler/kphp2cpp.cpp

```diff
@@ -221,6 +221,8 @@ int main(int argc, char *argv[]) {
              'l', "link-with", "KPHP_LINK_FILE");
   parser.add("Build runtime from sources", settings->force_link_runtime,
              "force-link-runtime", "KPHP_FORCE_LINK_RUNTIME");
+  parser.add("Enable safe compilation mode to forbid builtins stubs using", settings->forbid_stubs_using,
```

Comment:
`Enable safe compilation mode that treats calls to stub functions as compilation error`